### ISSUE TITLE
Combination improvement on the new product page

### ISSUE THREAD

**Issue by MatShir** [PrestaShop/PrestaShop#31588]
### Combination list

- [x]  Remove the reset mode because it is redundant with the cancel mode which causes exactly the same number of clicks (already done I believe)
- [x]  Display the edition mode at first click, and not at the blur (it was done on change, not on first click no reason to turn the edition mode until something has actually been edited no?)
- [x] On the edition mode, out-of-scope parameter can be hidden (need to search what else should be hidden/disabled: footer, tabs, ... To be experimented)

### Combination generation

- [x]  Close the modal after generation

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/43613217/222147757-94a8389a-c7fd-4121-9f59-dcc7cabf4716.png">

Acceptance test:When I click on generate combination, the modal close itself at the ending of the generation

### Combination modal

- [x]  Hide the history panel by default and displays it only after at least 2 pages edit

- [x] Adapt the Cancel button

When the modal has been edited
The button to close the modal is called 'Cancel'

When the modal has not been edited
The button to close the modal is called 'Close '

### Additional checks

- [x] https://github.com/PrestaShop/PrestaShop/issues/31660
- [x] https://github.com/PrestaShop/PrestaShop/issues/28934
- [x] https://github.com/PrestaShop/PrestaShop/issues/31417
- [x] https://github.com/PrestaShop/PrestaShop/issues/31424

**Comment by WahbiPS** [comment]
Fixed with this PR https://github.com/PrestaShop/PrestaShop/pull/31968
